source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.js

```diff
@@ -313,58 +338,73 @@ export default class Step1ConfigurationDialog extends Component<Props> {
         fullSize
       >
         <div className={styles.component}>
-          <p className={styles.description}>
-            <FormattedMessage
-              {...messages.description1}
-              values={{
-                itnLink,
-              }}
-            />{' '}
-            <FormattedHTMLMessage {...messages.description2} />
-          </p>
-          <Autocomplete
-            {...recoveryPhraseField.bind()}
-            ref={autocomplete => {
-              this.recoveryPhraseAutocomplete = autocomplete;
-            }}
-            options={suggestedMnemonics}
-            maxSelections={ITN_WALLET_RECOVERY_PHRASE_WORD_COUNT}
-            error={recoveryPhraseField.error}
-            maxVisibleOptions={5}
-            noResultsMessage={intl.formatMessage(messages.noResults)}
-            className={styles.recoveryPhrase}
-            skin={AutocompleteSkin}
-            optionHeight={50}
-            preselectedOptions={[...(recoveryPhrase || [])]}
-          />
-          <div className={styles.walletsDropdownWrapper}>
-            <WalletsDropdown
-              className={styles.walletsDropdown}
-              {...walletsDropdownField.bind()}
-              numberOfStakePools={4}
-              wallets={wallets}
-              onChange={onSelectWallet}
-              placeholder={intl.formatMessage(
-                messages.selectWalletInputPlaceholder
+          {!isSynced ? (
+            <p className={styles.redemptionUnavailable}>
```

Comment:
I'd move this into a dedicated component, like the **NoWalletsDialog.js**, as you're adding a lot of information that is actually not related to the Step 1 - Configuration.

Reply:
@daniloprates please see https://github.com/input-output-hk/daedalus/pull/2146#discussion_r468632031